src/dynamic_policy_service/dynamic.policy.js

```diff
@@ -0,0 +1,165 @@
+"use strict";
+
+const constants = require("../yoti_common/constants");
+
+const validateWanted = (wanted) => {
+    if (!Array.isArray(wanted)) {
+        return false;
+    }
+    const validAttributes = new Set([
+        "name",
+        "accept_self_asserted",
+        "derivation",
+        "optional",
+    ]);
+
+    for (const item of wanted) {
+        for (const key of Object.keys(item)) {
+            if (!validAttributes.has(key)) {
+                throw new Error(`invalid key value attribute supplied ${key}`);
+            }
+        }
+    }
+    return null;
+}
+
+/**
+ * Verify the type of remember-me.
```

Comment:
We refer to this as `RememberMeID` in the documentation and in the SDKs themselves. Looks like you call it `remember-me`/`rememberMe` throughout, if you could change the occurrences to `RememeberMeId` that would be great 👍

Reply:
Sorry Alttaf, I was looking at [Mike's Java PR for dynamic scenarios](https://github.com/getyoti/yoti-java-sdk/pull/58/files), and it made me realise that this `RememberMe` is with respect to the policy, not the rememberMeID of the user, apologies! So we only want to change the instances where it would be relating to the User, don't know if there are any of those.